rxn4chemistry/core.py

```diff
@@ -1158,3 +1147,112 @@ def update_roborxn_api_key(
         )
 
         return response
+
+    @response_handling(success_status_code=200, on_success=reaction_settings_on_success)
+    @ibm_rxn_api_limits
+    def get_reaction_settings(
+         self, synthesis_id: str, node_id: str
+    ) -> requests.models.Response:
+        """
+        Retrieves a the actions and product from a specified synthesis and node. 
+        Args:
+            synthesis_id (str): a synthesis identifier returned by
+                create_synthesis_from_sequence() method.
+            node_id (str): the 'id' field of a specific node in the synthesis tree
+
+        Returns:
+            actions (List[Dict[str, Any]): A list of actions which will completely replace
+                the existing actions for this node.
+            products (List[Dict[str, Any]): A list of product which will completely replace
+                the existing product for this node.
+        Examples:
+            Getting the actions and product of a synthesis.
+            >>> actions, product = rxn4chemistry_wrapper.get_reaction_settings(
+                            synthesis_id='5dd273618sid4897af',
+                            node_id='5z7f6bgz6g95gcbh')
+        """
+
+        response = requests.get(
+            self.routes.synthesis_reaction_setting_url.format(
+                synthesis_id=synthesis_id, node_id=node_id
+            ),
+            headers=self.headers,
+            cookies={},
+        )
+
+        return response
+
+    @response_handling(success_status_code=200, on_success=default_on_success)
+    @ibm_rxn_api_limits
+    def update_reaction_settings(
+         self, synthesis_id: str, node_id: str, actions: List[Dict[str, Any]], product: List[Dict[str, Any]]
+    ) -> requests.models.Response:
+        """
+        Update the list of actions and products for a specific node. The given actions will completely replace
+        the existing actions for this node in the tree. The given products will completely replace
+        the existing product for this node in the tree
+
+        Args:
+            synthesis_id (str): a synthesis identifier returned by
+                create_synthesis_from_sequence() method.
+            node_id (str): the 'id' field of a specific node in the synthesis tree
+            actions (List[Dict[str, Any]): A list of actions which will completely replace
+                the existing actions for this node.
+            products (List[Dict[str, Any]): A list of product which will completely replace
```

Comment:
if the typing changes update the types as well:
```suggestion
            product (Dict[str, Any]): A list of product which will completely replace
```